### DISCUSSION TITLE
Why is not possible to change Next/Previous tab hotkeys?

### DISCUSSION THREAD

**Discussion by VaclavObornik** [cmderdev/cmder#2444] [General]
Hello,

is there any reason, why the Next and Previous tab hotkeys are labeled as system and its keyboard shortcuts cannot be changed?
Would it be possible to free the limitation?

Thank you!

**Comment by daxgames** [comment]
Thats a question for the Conemu developer.  Cmder redistributes Conemu as its terminal emulator.

**Comment by mrx23dot** [comment]
ctrl+tab, ctrl+T, ctrl+W, ctrl+1, ctrl+2
like browser
